elastic/kibana
# [Fleet] Display featured integrations at top of integrations list per category

## Summary
Part of https://github.com/elastic/kibana/issues/141823.

Note: this functionality is behind a feature flag until https://github.com/elastic/integrations/pull/4392 is merged

Instead of showing large featured integrations at the top of the page, promote the featured integrations to the top of the search results but display them the same. Featured integrations are per category:

- no category selected: APM, Endpoint and Web Crawler should be the top integrations
- security category selected: cloud posture, network traffic and endpoint should be top integrations (Wont work until https://github.com/elastic/integrations/pull/4392 merged)

<img width="1264" alt="Screenshot 2022-10-10 at 15 27 02" src="https://user-images.githubusercontent.com/3315046/194889159-5df34eb8-d09a-48b7-9430-31813736e7a8.png">


### Testing steps

- 1. Add the following to your kibana config:

```
xpack.fleet.enableExperimental: 
    - noLargeFeaturedIntegrations
```
- 2. Browse integrations

- 3.  With no category selected: APM, Endpoint and Web Crawler should be the top integrations

- 4. With security category selected: cloud posture, network traffic and endpoint should be top integrations (Wont work until https://github.com/elastic/integrations/pull/4392 merged)

### Checklist

Delete any items that are not applicable to this PR.
